selenium_tests/manage_sections/page_objects/manage_sections_main_page.py

```diff
@@ -0,0 +1,16 @@
+"""
+This file models the Manage Sections Main (Landing) Page
+"""
+
+from selenium.webdriver.common.by import By
+from selenium_tests.manage_sections.page_objects\
+    .manage_sections_base_page_object import ManageSectionsBasePageObject
+
+
+class Locators(object):
+
+    MANAGE_SECTIONS_TEXT = (By.XPATH, './/h3[contains(., "Manage Sections")]')
```

Comment:
This will match all of the pages in manage sections, not just the main page, right?